src/Dashboard/Types/NodesStatistics/Statistic.php

```diff
@@ -190,8 +201,10 @@ protected function prepareChartData()
      * @param int $length
      * @return string
      */
-    private static function randomString($length = 16) {
+    private static function randomString($length = 16)
```

Comment:
But this is a Laravel package we are going to publish to the community. If we're going create a static method to generate a random string when Laravel has one with argument a frontend developer didn't want numbers in it, people are going to laugh.

We are going to make stuff like this we need to reevaluate our vision and wishes of being "pro laravel developers" in the community. This is definitely the way :)

But that's just my 50 cents.